server/boot/user.js

```diff
@@ -324,7 +324,7 @@ module.exports = function(app) {
     // not of the profile she is viewing
     const timezone = user && user.timezone ?
       user.timezone :
-      'EST';
+      moment.tz.guess();
```

Comment:
Does moment come with the tz package by default? I seem to remember we wanted to avoid this package. The reason escapes me at the moment (pun intended).

Reply:
The moment var is actually 'moment-timezone' so I don't see why tz won't come with it. I'm not sure though. :)

Reply:
What does this change accomplish?

Reply:
Hopefully a smarter guess on timezone than a fixed default. Haha. Idk if it somehow can get the user timezone.

Reply:
[`guess`](http://momentjs.com/timezone/docs/#/using-timezones/guessing-user-timezone/) uses browser APIs or the `Date` object to guess the timezone of the machine it's running on. It will have no way of getting the timezone of a user making the request from a separate machine.

Reply:
Right that makes sense. Haha. Closing this now.